examples/async-sign/src/lib.rs

```diff
@@ -31,7 +31,7 @@ pub struct InputArgs {
 gstd::metadata! {
     title: "demo async sign",
     init:
-        input: InputArgs,
```

Comment:
For me, these tabs increase readability of macro

Reply:
sry my editor formated this by default